packages/reports/src/Containers/open-positions.tsx

```diff
@@ -480,7 +484,10 @@ const OpenPositions = ({
     const [accumulator_rate, setAccumulatorRate] = React.useState(accumulator_rates[0]);
     const is_accumulator_selected = contract_type_value === contract_types[2].text;
     const is_multiplier_selected = contract_type_value === contract_types[1].text;
-    const contract_types_list = contract_types.map(({ text }) => ({ text, value: text }));
+    const show_accu_in_dropdown = !is_eu && is_virtual;
```

Comment:
I assume this means to show Accumulators only in Demo account because currently it's not available for real accounts. Is there a way to show it dynamically, without needing to change this logic again when accumulators will be launched on real accounts?

Reply:
let me look into it. Currently its mapped from `contract_types` which is hard coded. I looked into it previously but maybe now it is possible.

Reply:
not possibile at the moment.